Did some merge or something go wrong? These are errors that have to do with another PR

> C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(28,15): error CS1061: 'IPlatformElementConfiguration<iOS, Page>' does not contain a definition for 'SetModalPopoverView' and no accessible extension method 'SetModalPopoverView' accepting a first argument of type 'IPlatformElementConfiguration<iOS, Page>' could be found (are you missing a using directive or an assembly reference?) [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(33,15): error CS1061: 'IPlatformElementConfiguration<iOS, Page>' does not contain a definition for 'SetModalPopoverRect' and no accessible extension method 'SetModalPopoverRect' accepting a first argument of type 'IPlatformElementConfiguration<iOS, Page>' could be found (are you missing a using directive or an assembly reference?) [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(49,101): error CS0117: 'UIModalPresentationStyle' does not contain a definition for 'Popover' [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(56,101): error CS0117: 'UIModalPresentationStyle' does not contain a definition for 'Popover' [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(28,15): error CS1061: 'IPlatformElementConfiguration<iOS, Page>' does not contain a definition for 'SetModalPopoverView' and no accessible extension method 'SetModalPopoverView' accepting a first argument of type 'IPlatformElementConfiguration<iOS, Page>' could be found (are you missing a using directive or an assembly reference?) [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-windows10.0.19041.0]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(33,15): error CS1061: 'IPlatformElementConfiguration<iOS, Page>' does not contain a definition for 'SetModalPopoverRect' and no accessible extension method 'SetModalPopoverRect' accepting a first argument of type 'IPlatformElementConfiguration<iOS, Page>' could be found (are you missing a using directive or an assembly reference?) [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-windows10.0.19041.0]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(49,101): error CS0117: 'UIModalPresentationStyle' does not contain a definition for 'Popover' [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-windows10.0.19041.0]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(56,101): error CS0117: 'UIModalPresentationStyle' does not contain a definition for 'Popover' [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-windows10.0.19041.0]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(28,15): error CS1061: 'IPlatformElementConfiguration<iOS, Page>' does not contain a definition for 'SetModalPopoverView' and no accessible extension method 'SetModalPopoverView' accepting a first argument of type 'IPlatformElementConfiguration<iOS, Page>' could be found (are you missing a using directive or an assembly reference?) [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-maccatalyst]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(33,15): error CS1061: 'IPlatformElementConfiguration<iOS, Page>' does not contain a definition for 'SetModalPopoverRect' and no accessible extension method 'SetModalPopoverRect' accepting a first argument of type 'IPlatformElementConfiguration<iOS, Page>' could be found (are you missing a using directive or an assembly reference?) [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-maccatalyst]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(49,101): error CS0117: 'UIModalPresentationStyle' does not contain a definition for 'Popover' [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-maccatalyst]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(56,101): error CS0117: 'UIModalPresentationStyle' does not contain a definition for 'Popover' [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-maccatalyst]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(28,15): error CS1061: 'IPlatformElementConfiguration<iOS, Page>' does not contain a definition for 'SetModalPopoverView' and no accessible extension method 'SetModalPopoverView' accepting a first argument of type 'IPlatformElementConfiguration<iOS, Page>' could be found (are you missing a using directive or an assembly reference?) [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-ios]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(33,15): error CS1061: 'IPlatformElementConfiguration<iOS, Page>' does not contain a definition for 'SetModalPopoverRect' and no accessible extension method 'SetModalPopoverRect' accepting a first argument of type 'IPlatformElementConfiguration<iOS, Page>' could be found (are you missing a using directive or an assembly reference?) [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-ios]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(49,101): error CS0117: 'UIModalPresentationStyle' does not contain a definition for 'Popover' [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-ios]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(56,101): error CS0117: 'UIModalPresentationStyle' does not contain a definition for 'Popover' [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-ios]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(28,15): error CS1061: 'IPlatformElementConfiguration<iOS, Page>' does not contain a definition for 'SetModalPopoverView' and no accessible extension method 'SetModalPopoverView' accepting a first argument of type 'IPlatformElementConfiguration<iOS, Page>' could be found (are you missing a using directive or an assembly reference?) [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-windows10.0.20348.0]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(33,15): error CS1061: 'IPlatformElementConfiguration<iOS, Page>' does not contain a definition for 'SetModalPopoverRect' and no accessible extension method 'SetModalPopoverRect' accepting a first argument of type 'IPlatformElementConfiguration<iOS, Page>' could be found (are you missing a using directive or an assembly reference?) [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-windows10.0.20348.0]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(49,101): error CS0117: 'UIModalPresentationStyle' does not contain a definition for 'Popover' [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-windows10.0.20348.0]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(56,101): error CS0117: 'UIModalPresentationStyle' does not contain a definition for 'Popover' [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-windows10.0.20348.0]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(28,15): error CS1061: 'IPlatformElementConfiguration<iOS, Page>' does not contain a definition for 'SetModalPopoverView' and no accessible extension method 'SetModalPopoverView' accepting a first argument of type 'IPlatformElementConfiguration<iOS, Page>' could be found (are you missing a using directive or an assembly reference?) [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-android]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(33,15): error CS1061: 'IPlatformElementConfiguration<iOS, Page>' does not contain a definition for 'SetModalPopoverRect' and no accessible extension method 'SetModalPopoverRect' accepting a first argument of type 'IPlatformElementConfiguration<iOS, Page>' could be found (are you missing a using directive or an assembly reference?) [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-android]
 C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Pages\PlatformSpecifics\iOS\iOSModalPagePresentationStyle.xaml.cs(49,101): error CS0117: 'UIModalPresentationStyle' does not contain a definition for 'Popover' [C:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj::TargetFramework=net10.0-android]

--- a/src/Essentials/src/Geolocation/Geolocation.android.cs
+++ b/src/Essentials/src/Geolocation/Geolocation.android.cs
@@ -30,6 +30,19 @@ namespace Microsoft.Maui.Devices.Sensors
 		/// </summary>
 		public bool IsListeningForeground { get => continuousListener is not null; }
 
+		public bool IsSupported
+		{
+			get
+			{
+				if (LocationManager is null)
+					return false;
+
+				return OperatingSystem.IsAndroidVersionAtLeast(28)
+					? LocationManager.IsLocationEnabled
+					: LocationManager.IsProviderEnabled(LocationManager.GpsProvider) || LocationManager.IsProviderEnabled(LocationManager.NetworkProvider);
+			}
+		}
+
 		public async Task<Location?> GetLastKnownLocationAsync()
 		{
 			if (LocationManager is null)

--- a/src/Essentials/src/Geolocation/Geolocation.android.cs
+++ b/src/Essentials/src/Geolocation/Geolocation.android.cs
@@ -30,7 +30,7 @@ namespace Microsoft.Maui.Devices.Sensors
 		/// </summary>
 		public bool IsListeningForeground { get => continuousListener is not null; }
 
-		public bool IsSupported
+		public bool IsEnabled
 		{
 			get
 			{

--- a/src/Essentials/src/Geolocation/Geolocation.ios.macos.cs
+++ b/src/Essentials/src/Geolocation/Geolocation.ios.macos.cs
@@ -18,11 +18,11 @@ namespace Microsoft.Maui.Devices.Sensors
 		/// </summary>
 		public bool IsListeningForeground { get => listeningManager != null; }
 
-		public bool IsSupported => CLLocationManager.LocationServicesEnabled;
+		public bool IsEnabled => CLLocationManager.LocationServicesEnabled;
 
 		public async Task<Location?> GetLastKnownLocationAsync()
 		{
-			if (!IsSupported)
+			if (!IsEnabled)
 				throw new FeatureNotEnabledException("Location services are not enabled on device.");
 
 			await Permissions.EnsureGrantedAsync<Permissions.LocationWhenInUse>();
@@ -44,7 +44,7 @@ namespace Microsoft.Maui.Devices.Sensors
 		{
 			ArgumentNullException.ThrowIfNull(request);
 
-			if (!IsSupported)
+			if (!IsEnabled)
 				throw new FeatureNotEnabledException("Location services are not enabled on device.");
 
 			await Permissions.EnsureGrantedAsync<Permissions.LocationWhenInUse>();
@@ -121,7 +121,7 @@ namespace Microsoft.Maui.Devices.Sensors
 			if (IsListeningForeground)
 				throw new InvalidOperationException("Already listening to location changes.");
 
-			if (!IsSupported)
+			if (!IsEnabled)
 				throw new FeatureNotEnabledException("Location services are not enabled on device.");
 
 			await Permissions.EnsureGrantedAsync<Permissions.LocationWhenInUse>();

--- a/src/Essentials/src/Geolocation/Geolocation.ios.macos.cs
+++ b/src/Essentials/src/Geolocation/Geolocation.ios.macos.cs
@@ -18,9 +18,11 @@ namespace Microsoft.Maui.Devices.Sensors
 		/// </summary>
 		public bool IsListeningForeground { get => listeningManager != null; }
 
+		public bool IsSupported => CLLocationManager.LocationServicesEnabled;
+
 		public async Task<Location?> GetLastKnownLocationAsync()
 		{
-			if (!CLLocationManager.LocationServicesEnabled)
+			if (!IsSupported)
 				throw new FeatureNotEnabledException("Location services are not enabled on device.");
 
 			await Permissions.EnsureGrantedAsync<Permissions.LocationWhenInUse>();
@@ -42,7 +44,7 @@ namespace Microsoft.Maui.Devices.Sensors
 		{
 			ArgumentNullException.ThrowIfNull(request);
 
-			if (!CLLocationManager.LocationServicesEnabled)
+			if (!IsSupported)
 				throw new FeatureNotEnabledException("Location services are not enabled on device.");
 
 			await Permissions.EnsureGrantedAsync<Permissions.LocationWhenInUse>();
@@ -119,7 +121,7 @@ namespace Microsoft.Maui.Devices.Sensors
 			if (IsListeningForeground)
 				throw new InvalidOperationException("Already listening to location changes.");
 
-			if (!CLLocationManager.LocationServicesEnabled)
+			if (!IsSupported)
 				throw new FeatureNotEnabledException("Location services are not enabled on device.");
 
 			await Permissions.EnsureGrantedAsync<Permissions.LocationWhenInUse>();

--- a/src/Essentials/src/Geolocation/Geolocation.netstandard.tvos.watchos.cs
+++ b/src/Essentials/src/Geolocation/Geolocation.netstandard.tvos.watchos.cs
@@ -16,7 +16,7 @@ namespace Microsoft.Maui.Devices.Sensors
 
 		public bool IsListeningForeground { get => false; }
 
-		public bool IsSupported { get => false; }
+		public bool IsEnabled { get => false; }
 
 		public Task<bool> StartListeningForegroundAsync(GeolocationListeningRequest request) =>
 			throw ExceptionUtils.NotSupportedOrImplementedException;

--- a/src/Essentials/src/Geolocation/Geolocation.netstandard.tvos.watchos.cs
+++ b/src/Essentials/src/Geolocation/Geolocation.netstandard.tvos.watchos.cs
@@ -16,6 +16,8 @@ namespace Microsoft.Maui.Devices.Sensors
 
 		public bool IsListeningForeground { get => false; }
 
+		public bool IsSupported { get => false; }
+
 		public Task<bool> StartListeningForegroundAsync(GeolocationListeningRequest request) =>
 			throw ExceptionUtils.NotSupportedOrImplementedException;
 

--- a/src/Essentials/src/Geolocation/Geolocation.shared.cs
+++ b/src/Essentials/src/Geolocation/Geolocation.shared.cs
@@ -36,9 +36,9 @@ namespace Microsoft.Maui.Devices.Sensors
 		bool IsListeningForeground { get; }
 
 		/// <summary>
-		/// Returns true when the device's location services are supported
+		/// Returns true when the device's location services are enabled
 		/// </summary>
-		bool IsSupported { get; }
+		bool IsEnabled { get; }
 
 		/// <summary>
 		/// Occurs while listening to location updates.
@@ -120,9 +120,9 @@ namespace Microsoft.Maui.Devices.Sensors
 		public static bool IsListeningForeground { get => Current.IsListeningForeground; }
 
 		/// <summary>
-		/// Returns true when the device's location services are supported
+		/// Returns true when the device's location services are enabled
 		/// </summary>
-		public static bool IsSupported { get => Current.IsSupported; }
+		public static bool IsEnabled { get => Current.IsEnabled; }
 
 		/// <summary>
 		/// Occurs while listening to location updates.

--- a/src/Essentials/src/Geolocation/Geolocation.shared.cs
+++ b/src/Essentials/src/Geolocation/Geolocation.shared.cs
@@ -36,6 +36,11 @@ namespace Microsoft.Maui.Devices.Sensors
 		bool IsListeningForeground { get; }
 
 		/// <summary>
+		/// Returns true when the device's location services are supported
+		/// </summary>
+		bool IsSupported { get; }
+
+		/// <summary>
 		/// Occurs while listening to location updates.
 		/// </summary>
 		event EventHandler<GeolocationLocationChangedEventArgs>? LocationChanged;
@@ -113,6 +118,11 @@ namespace Microsoft.Maui.Devices.Sensors
 		/// Indicates if currently listening to location updates while the app is in foreground.
 		/// </summary>
 		public static bool IsListeningForeground { get => Current.IsListeningForeground; }
+
+		/// <summary>
+		/// Returns true when the device's location services are supported
+		/// </summary>
+		public static bool IsSupported { get => Current.IsSupported; }
 
 		/// <summary>
 		/// Occurs while listening to location updates.

--- a/src/Essentials/src/Geolocation/Geolocation.tizen.cs
+++ b/src/Essentials/src/Geolocation/Geolocation.tizen.cs
@@ -13,7 +13,7 @@ namespace Microsoft.Maui.Devices.Sensors
 
 		public bool IsListeningForeground { get => false; }
 		
-		public bool IsSupported { get => false; }
+		public bool IsEnabled { get => false; }
 
 		public Task<Location?> GetLastKnownLocationAsync() => Task.FromResult<Location?>(lastKnownLocation);
 

--- a/src/Essentials/src/Geolocation/Geolocation.tizen.cs
+++ b/src/Essentials/src/Geolocation/Geolocation.tizen.cs
@@ -12,6 +12,8 @@ namespace Microsoft.Maui.Devices.Sensors
 		Location lastKnownLocation = new Location();
 
 		public bool IsListeningForeground { get => false; }
+		
+		public bool IsSupported { get => false; }
 
 		public Task<Location?> GetLastKnownLocationAsync() => Task.FromResult<Location?>(lastKnownLocation);
 

--- a/src/Essentials/src/Geolocation/Geolocation.uwp.cs
+++ b/src/Essentials/src/Geolocation/Geolocation.uwp.cs
@@ -16,6 +16,15 @@ namespace Microsoft.Maui.Devices.Sensors
 		/// </summary>
 		public bool IsListeningForeground { get => listeningGeolocator != null; }
 
+		public bool IsSupported
+		{
+			get
+			{
+				var geolocator = new Geolocator();
+				return geolocator.LocationStatus != PositionStatus.Disabled && geolocator.LocationStatus != PositionStatus.NotAvailable;
+			}
+		}
+
 		public async Task<Location?> GetLastKnownLocationAsync()
 		{
 			// no need for permissions as AllowFallbackToConsentlessPositions

--- a/src/Essentials/src/Geolocation/Geolocation.uwp.cs
+++ b/src/Essentials/src/Geolocation/Geolocation.uwp.cs
@@ -16,7 +16,7 @@ namespace Microsoft.Maui.Devices.Sensors
 		/// </summary>
 		public bool IsListeningForeground { get => listeningGeolocator != null; }
 
-		public bool IsSupported
+		public bool IsEnabled
 		{
 			get
 			{

--- a/src/Essentials/src/PublicAPI/net-android/PublicAPI.Unshipped.txt
+++ b/src/Essentials/src/PublicAPI/net-android/PublicAPI.Unshipped.txt
@@ -1,4 +1,6 @@
 #nullable enable
+Microsoft.Maui.Devices.Sensors.IGeolocation.IsSupported.get -> bool
+static Microsoft.Maui.Devices.Sensors.Geolocation.IsSupported.get -> bool
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue, string? sharedName) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Set(string! key, System.DateTimeOffset value, string? sharedName) -> void

--- a/src/Essentials/src/PublicAPI/net-android/PublicAPI.Unshipped.txt
+++ b/src/Essentials/src/PublicAPI/net-android/PublicAPI.Unshipped.txt
@@ -1,6 +1,6 @@
 #nullable enable
-Microsoft.Maui.Devices.Sensors.IGeolocation.IsSupported.get -> bool
-static Microsoft.Maui.Devices.Sensors.Geolocation.IsSupported.get -> bool
+Microsoft.Maui.Devices.Sensors.IGeolocation.IsEnabled.get -> bool
+static Microsoft.Maui.Devices.Sensors.Geolocation.IsEnabled.get -> bool
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue, string? sharedName) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Set(string! key, System.DateTimeOffset value, string? sharedName) -> void

--- a/src/Essentials/src/PublicAPI/net-ios/PublicAPI.Unshipped.txt
+++ b/src/Essentials/src/PublicAPI/net-ios/PublicAPI.Unshipped.txt
@@ -1,4 +1,6 @@
 #nullable enable
+Microsoft.Maui.Devices.Sensors.IGeolocation.IsSupported.get -> bool
+static Microsoft.Maui.Devices.Sensors.Geolocation.IsSupported.get -> bool
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue, string? sharedName) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Set(string! key, System.DateTimeOffset value, string? sharedName) -> void

--- a/src/Essentials/src/PublicAPI/net-ios/PublicAPI.Unshipped.txt
+++ b/src/Essentials/src/PublicAPI/net-ios/PublicAPI.Unshipped.txt
@@ -1,6 +1,6 @@
 #nullable enable
-Microsoft.Maui.Devices.Sensors.IGeolocation.IsSupported.get -> bool
-static Microsoft.Maui.Devices.Sensors.Geolocation.IsSupported.get -> bool
+Microsoft.Maui.Devices.Sensors.IGeolocation.IsEnabled.get -> bool
+static Microsoft.Maui.Devices.Sensors.Geolocation.IsEnabled.get -> bool
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue, string? sharedName) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Set(string! key, System.DateTimeOffset value, string? sharedName) -> void

--- a/src/Essentials/src/PublicAPI/net-maccatalyst/PublicAPI.Unshipped.txt
+++ b/src/Essentials/src/PublicAPI/net-maccatalyst/PublicAPI.Unshipped.txt
@@ -1,4 +1,6 @@
 #nullable enable
+Microsoft.Maui.Devices.Sensors.IGeolocation.IsSupported.get -> bool
+static Microsoft.Maui.Devices.Sensors.Geolocation.IsSupported.get -> bool
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue, string? sharedName) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Set(string! key, System.DateTimeOffset value, string? sharedName) -> void

--- a/src/Essentials/src/PublicAPI/net-maccatalyst/PublicAPI.Unshipped.txt
+++ b/src/Essentials/src/PublicAPI/net-maccatalyst/PublicAPI.Unshipped.txt
@@ -1,6 +1,6 @@
 #nullable enable
-Microsoft.Maui.Devices.Sensors.IGeolocation.IsSupported.get -> bool
-static Microsoft.Maui.Devices.Sensors.Geolocation.IsSupported.get -> bool
+Microsoft.Maui.Devices.Sensors.IGeolocation.IsEnabled.get -> bool
+static Microsoft.Maui.Devices.Sensors.Geolocation.IsEnabled.get -> bool
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue, string? sharedName) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Set(string! key, System.DateTimeOffset value, string? sharedName) -> void

--- a/src/Essentials/src/PublicAPI/net-tizen/PublicAPI.Unshipped.txt
+++ b/src/Essentials/src/PublicAPI/net-tizen/PublicAPI.Unshipped.txt
@@ -1,4 +1,6 @@
 #nullable enable
+Microsoft.Maui.Devices.Sensors.IGeolocation.IsSupported.get -> bool
+static Microsoft.Maui.Devices.Sensors.Geolocation.IsSupported.get -> bool
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue, string? sharedName) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Set(string! key, System.DateTimeOffset value, string? sharedName) -> void

--- a/src/Essentials/src/PublicAPI/net-tizen/PublicAPI.Unshipped.txt
+++ b/src/Essentials/src/PublicAPI/net-tizen/PublicAPI.Unshipped.txt
@@ -1,6 +1,6 @@
 #nullable enable
-Microsoft.Maui.Devices.Sensors.IGeolocation.IsSupported.get -> bool
-static Microsoft.Maui.Devices.Sensors.Geolocation.IsSupported.get -> bool
+Microsoft.Maui.Devices.Sensors.IGeolocation.IsEnabled.get -> bool
+static Microsoft.Maui.Devices.Sensors.Geolocation.IsEnabled.get -> bool
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue, string? sharedName) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Set(string! key, System.DateTimeOffset value, string? sharedName) -> void

--- a/src/Essentials/src/PublicAPI/net-windows/PublicAPI.Unshipped.txt
+++ b/src/Essentials/src/PublicAPI/net-windows/PublicAPI.Unshipped.txt
@@ -1,4 +1,6 @@
 #nullable enable
+Microsoft.Maui.Devices.Sensors.IGeolocation.IsSupported.get -> bool
+static Microsoft.Maui.Devices.Sensors.Geolocation.IsSupported.get -> bool
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue, string? sharedName) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Set(string! key, System.DateTimeOffset value, string? sharedName) -> void

--- a/src/Essentials/src/PublicAPI/net-windows/PublicAPI.Unshipped.txt
+++ b/src/Essentials/src/PublicAPI/net-windows/PublicAPI.Unshipped.txt
@@ -1,6 +1,6 @@
 #nullable enable
-Microsoft.Maui.Devices.Sensors.IGeolocation.IsSupported.get -> bool
-static Microsoft.Maui.Devices.Sensors.Geolocation.IsSupported.get -> bool
+Microsoft.Maui.Devices.Sensors.IGeolocation.IsEnabled.get -> bool
+static Microsoft.Maui.Devices.Sensors.Geolocation.IsEnabled.get -> bool
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue, string? sharedName) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Set(string! key, System.DateTimeOffset value, string? sharedName) -> void

--- a/src/Essentials/src/PublicAPI/net/PublicAPI.Unshipped.txt
+++ b/src/Essentials/src/PublicAPI/net/PublicAPI.Unshipped.txt
@@ -1,4 +1,6 @@
 #nullable enable
+Microsoft.Maui.Devices.Sensors.IGeolocation.IsSupported.get -> bool
+static Microsoft.Maui.Devices.Sensors.Geolocation.IsSupported.get -> bool
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue, string? sharedName) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Set(string! key, System.DateTimeOffset value, string? sharedName) -> void

--- a/src/Essentials/src/PublicAPI/net/PublicAPI.Unshipped.txt
+++ b/src/Essentials/src/PublicAPI/net/PublicAPI.Unshipped.txt
@@ -1,6 +1,6 @@
 #nullable enable
-Microsoft.Maui.Devices.Sensors.IGeolocation.IsSupported.get -> bool
-static Microsoft.Maui.Devices.Sensors.Geolocation.IsSupported.get -> bool
+Microsoft.Maui.Devices.Sensors.IGeolocation.IsEnabled.get -> bool
+static Microsoft.Maui.Devices.Sensors.Geolocation.IsEnabled.get -> bool
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue, string? sharedName) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Set(string! key, System.DateTimeOffset value, string? sharedName) -> void

--- a/src/Essentials/src/PublicAPI/netstandard/PublicAPI.Unshipped.txt
+++ b/src/Essentials/src/PublicAPI/netstandard/PublicAPI.Unshipped.txt
@@ -1,4 +1,6 @@
 #nullable enable
+Microsoft.Maui.Devices.Sensors.IGeolocation.IsSupported.get -> bool
+static Microsoft.Maui.Devices.Sensors.Geolocation.IsSupported.get -> bool
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue, string? sharedName) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Set(string! key, System.DateTimeOffset value, string? sharedName) -> void

--- a/src/Essentials/src/PublicAPI/netstandard/PublicAPI.Unshipped.txt
+++ b/src/Essentials/src/PublicAPI/netstandard/PublicAPI.Unshipped.txt
@@ -1,6 +1,6 @@
 #nullable enable
-Microsoft.Maui.Devices.Sensors.IGeolocation.IsSupported.get -> bool
-static Microsoft.Maui.Devices.Sensors.Geolocation.IsSupported.get -> bool
+Microsoft.Maui.Devices.Sensors.IGeolocation.IsEnabled.get -> bool
+static Microsoft.Maui.Devices.Sensors.Geolocation.IsEnabled.get -> bool
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue, string? sharedName) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Get(string! key, System.DateTimeOffset defaultValue) -> System.DateTimeOffset
 static Microsoft.Maui.Storage.Preferences.Set(string! key, System.DateTimeOffset value, string? sharedName) -> void

